tools/websocket_to_posix_proxy/CMakeLists.txt

```
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 2.8)
+
+project(websocket_to_posix_proxy)
+
+file(GLOB sourceFiles src/*.cpp src/*.h)
+
+add_executable(websocket_to_posix_proxy ${sourceFiles})
+
```

Running the tests, I encountered:

> [100%] Linking CXX executable websocket_to_posix_proxy
/usr/bin/ld: CMakeFiles/websocket_to_posix_proxy.dir/src/main.cpp.o: in function `main':
main.cpp:(.text+0x1144): undefined reference to `pthread_create'

On Linux, this fixed the issue.

```suggestion
find_package(Threads)
target_link_libraries(websocket_to_posix_proxy ${CMAKE_THREAD_LIBS_INIT})
```